docs/guidelines/outline.md

```diff
@@ -8,36 +8,58 @@ This style guide provides instruction for translating to the [Your locale name]
 
 	Include other relevant introductory information here, such as a community webpage or contact information.
 
-## Formality and Tone
+## Formality and tone
 
-	Who is the target user for this project and what is their background?
+	Who is the target user for this project and what is their background? Describe briefly the users from your locale. If the type of user from one project to another will vary, indicate so here.
+
+		Describe briefly the users from your locale. If the type of user from one project to another will vary, indicate so here.
```

Comment:
I'm not sure I understand why this sentence seems to be repeated twice in a row?
Also, the more I look at the doc, the more I wonder if the format is good for these subparts. I need to scroll horizontally a lot to see the entire sentences, for each paragraph. Is there a reason you wanted it to appear this way?

Reply:
I second you on the code blocks. I thought they were only displayed this way on GitHub (with a horizontal scroll), but just realized it’s also the case in Gitbook.

Maybe we could keep them only for actual examples? E.g Date format table, phone number example, address examples, etc.
They are short enough, so that there would be no horizontal scroll here.

Reply:
Yes, the duplicated sentence is probably an extra Ctrl+V I hit during moving the texts from one document to the other.

As for the code blocks, I just kept the formatting from the existing outline. For someone working with the copy and replacing the blocks with actual text there won't be any scrolling. However for someone reading it, blockquotes will be better. Let's replace the code blocks with blockquotes. Maybe that will fix the appearance of tables too. If not, I will just move them out of the blocks.